packages/seacas/libraries/ioss/src/main/CMakeLists.txt

```diff
@@ -380,6 +380,21 @@ TRIBITS_ADD_ADVANCED_TEST(structured_cgns_roundtrip_zgc
   COMM mpi serial
   )
 
+TRIBITS_ADD_ADVANCED_TEST(structured_cgns_periodic_roundtrip_fpp
+   TEST_0 EXEC io_shell ARGS --compose external ${CMAKE_CURRENT_SOURCE_DIR}/test/self-looping-grid.cgns self-looping-fpp.cgns
+     NUM_MPI_PROCS 1-4
```

Comment:
Is a range of procs allows `1-4`?  There must not be a check for that in the CMake code.  I think that this will build a command of the form:

```
mpiexec -np 1-4 ...
```  

That is not what you want, is it?

Reply:
It has been working that way in other tests for a few years...  I thought I got the original example from other Trilinos code.

Reply:
Let's watch the output when this runs on CDash and see what happens.

Reply:
@bartlettroscoe I always interpreted "1-4" as "the test may run on any number of MPI processes 1, 2, 3, or 4."  Tpetra has used that for as long as I've been working on Tpetra.